README.md

```diff
@@ -30,6 +30,8 @@ Configure using an API token from Circle
 ```ruby
 CircleCi.configure do |config|
   config.token = 'my-token'
+  config.host = 'https://circleci.com/api/v1'
```

Comment:
Thinking the host should just be the host without any relative path ie. `https://ci.mycompany.com`. This will require some changes to how the config object gets used to make http calls. It would be nice to set the host, if need be, and configure a version as well. Then the config object can have an instance method to get the built out URI to use when making API calls. Not sure if people are using enterprise with non port 80 but if that's the case it might need to be intelligently considered in the URI as well for non-standard HTTP port.